interface/usergroup/usergroup_admin.php

```diff
@@ -290,7 +290,7 @@
                   sqlStatement("insert into groups set name = '" . trim(formData('groupname')) .
                     "', user = '" . trim(formData('rumple')) . "'");
 
-                if (isset($phpgacl_location) && acl_check('admin', 'acl') && trim(formData('rumple'))) {
+                if (isset($phpgacl_location) && trim(formData('rumple'))) {
```

Comment:
remove the isset($phpgacl_location) from this if

Reply:
Done.